hangups/http_utils.py

```diff
@@ -46,6 +46,9 @@ def fetch(method, url, params=None, headers=None, cookies=None, data=None,
         else:
             error_msg = None
             break
+        finally:
+            if 'res' in locals():
+                res.close()
```

Comment:
_I updated the gist: fixed the typo ``timeout_finallly`` -> ``timeout_finally``_